src/editor/Core/Types.re

```diff
@@ -44,6 +44,28 @@ module Mode = {
     };
 };
 
+/**
```

Comment:
Following from our convo on Discord - we might have to think about this (as Oni2 will be managing the windows / views) - but not a blocker for now.

Reply:
Fair enough :+1: